node/forest_libp2p/src/behaviour.rs

```diff
@@ -253,4 +287,41 @@ impl ForestBehaviour {
     pub fn peers(&self) -> &HashSet<PeerId> {
         &self.peers
     }
+
+    /// Adds peer to bitswap peer set
+    pub fn connect(&mut self, peer_id: PeerId) {
+        self.bitswap.connect(peer_id);
+    }
+
+    /// Send a block to a peer over bitswap
+    pub fn send_block(
+        &mut self,
+        peer_id: &PeerId,
+        cid: Cid,
+        data: Box<[u8]>,
+    ) -> Result<(), Box<dyn Error>> {
+        let cid = cid.to_string();
+        log::debug!("send {}", cid);
```

Comment:
nit: can you import so we call these consistently? Also might be useful to include in the log message that you sent over bitswap since the module would just be `forest_libp2p::behaviour` so could be anything